app/javascript/components/dashboard-form/dashboard-form.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './dashboard-form.schema';
+// import MiqFormRenderer from '../../forms/data-driven-form';
+
+class DashboardForm extends Component {
```

Comment:
I would prefer using functional component with hooks (`useState`) to make the component more efficient. More info here: https://reactjs.org/docs/hooks-intro.html